client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py

```diff
@@ -68,15 +68,21 @@ def process(self, instance):
                 # Replace the frame number with '%04d'
                 path_to_frame = re.sub(r"\.\d+\.", ".%04d.", local_path)
 
-                data_to_update["sg_path_to_frames"] = path_to_frame
+                data_to_update |= {
+                    "sg_path_to_movie": path_to_frame,
```

Comment:
why? this could be replacing the `sg_path_to_movie` from the video representation if the iteration happens after?